src/game_api/memory.cpp

```diff
@@ -147,9 +149,15 @@ void recover_mem(std::string name, size_t addr)
         for (auto& it : original_memory[name])
             if (!addr || addr == it.address)
                 write_mem_prot(it.address, std::string_view{it.old_data, it.size}, it.prot_used);
+        edited_memory.erase(name);
```

Comment:
should be only erased when `addr == 0` (whole name, not just single addr)

Reply:
right